### PR TITLE
Census: model supports imported SC data

### PR DESCRIPTION
**Pull request by breville**
Our S3 bucket now has `state_cs_offerings/SC/2017-2018.csv` containing state data for SC.

We didn't actually get course codes for SC, just a list of schools offering some form of computer science, so we've decided to use `"unspecified"` as the course code in such situations.

Note: to seed this data on a local machine, run `rake seed:state_cs_offerings`.  To seed the real data from S3, ensure that `stub_school_data: false` appears in `locals.yml` first.  Note that the real data relies on the local machine also having other school information also seeded from S3, in which case running `rake seed:all` with the same `locals.yml` flag will seed everything necessary.

As a side note: seeding from S3 uses the content's `etag` to decide if there's new content to seed.  The seed will no-op silently if the `etag` matches the last ingested.  The last ingested values are in the `SeededS3Object` model.

### CODE REVIEW

**Comment by drewsamnick** [comment]
This looks good. The only thought I had was if it might be clearer to set the course code to "unspecified" in code rather than filling in that value in the data file. That makes it easier and clearer if we end up using the same pattern for other states that don't give us course-specific data.

File: dashboard/app/models/census/state_cs_offering.rb [hunk 1]
```diff
@@ -69,6 +72,9 @@ def self.get_courses(state_code, row_hash)
     when 'ID'
       # A column per CS course with a value of 'Y' if the course is offered.
       ['02204',	'03208', '10157'].select {|course| row_hash[course] == 'Y'}
+    when 'SC'
+      # One source per row
+      [row_hash['coursecode']]
```
Comment (drewsamnick) [inline dashboard/app/models/census/state_cs_offering.rb]: My thought was if we ought to change this from:
`[row_hash['coursecode']]`
to:
`[UNSPECIFIED_COURSE]`

With the `UNSPECIFIED_COURSE` constant defined to be "unspecified"

Comment (breville) [reply dashboard/app/models/census/state_cs_offering.rb]: Oh, I had wondered that too.  Will make the change.

Comment (mirlew) [reply dashboard/app/models/census/state_cs_offering.rb]: agree with drew

**Pull request merged**